adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.h

```diff
@@ -15,5 +15,15 @@
 #import <Foundation/Foundation.h>
 #import <GoogleMobileAds/GoogleMobileAds.h>
 
+typedef NS_ENUM(NSInteger, GADPangleErrorCode) {
+    //pangle mediation configuration did not contain a valid app id
+    GADPangleErrorMissingValidAppId = 1000,
+    //slot id is nill
+    GADPangleErrorSlotIdNil = 1001,
+    // Pangle SDK version is too low
+    GADPangleErrorVersionLow = 1002,
```

Comment:
Question: What does this SDK version is too low here mean?

Reply:
Updating Pangle SDK to the latest version should fix it.